components/commands/common/BUILD.gn

```diff
@@ -5,19 +5,38 @@
 
 import("//mojo/public/tools/bindings/mojom.gni")
 
+action("generate_command_metadata") {
+  command_files = [
+    "//chrome/app/chrome_command_ids.h",
+    "//brave/app/brave_command_ids.h",
```

Comment:
Hmm, we're not allowing chrome layer from components.
To do that, this component should depend on `//chrome/app:command_ids` and `//brave/app:command_ids`.

Reply:
Moved back to `brave//app` - it'd be nice if this could live in commands/common :cry: 

The C++ doesn't actually depend on the command files but the generator does, :shrug: